From: Brian Norris <briannorris@chromium.org>
Date: Fri, 2 Sep 2022 13:40:17 -0700
Subject: [PATCH] ec: Fix DEBUG "variable" in usb_updater Makefile

DEBUG is not the way to refer to a variable in a Makefile expression;
$(DEBUG) is.

Also, imitate the =1 check from extra/rma_reset/Makefile.

BUG=none
TEST=build

Change-Id: Ic71be497ca83041dfdedf63c10e7c74b38c05bc8
Signed-off-by: Brian Norris <briannorris@chromium.org>
---
 extra/usb_updater/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/extra/usb_updater/Makefile b/extra/usb_updater/Makefile
index 04adfff4dd38..f1ec4f8b0e46 100644
--- a/extra/usb_updater/Makefile
+++ b/extra/usb_updater/Makefile
@@ -19,10 +19,10 @@ CFLAGS  := -std=gnu99 \
         -Wredundant-decls \
         -Wmissing-declarations
 
-ifeq (DEBUG,)
-CFLAGS += -O3
-else
+ifeq ($(DEBUG),1)
 CFLAGS += -O0
+else
+CFLAGS += -O3
 endif
 
 #
